core/src/reporter/utils.ts

```diff
@@ -65,22 +65,24 @@ export async function sendTransaction({
   payload,
   gasLimit,
   value,
-  logger
+  logger,
+  nonce
 }: {
-  wallet
+  wallet: NonceManager
   to: string
   payload?: string
   gasLimit?: number | string
   value?: number | string | ethers.BigNumber
   logger: Logger
+  nonce: number
```

Comment:
Nonce can be optional parameter here

Reply:
i think making nonce optional will introduce a potential risk to the whole service. If nonce is optional then:
 - we need to implement a fallback logic which is to use wallet's transactionCount, in which case
 - we might run into all sorts of nonce related errors since we are managing it manually and the `wallet.transactionCount()` is almost always behind our local nonce value

Reply:
`sendTransaction`,  `sendTransactionDelegatedFee`, `sendTransactionCaver` can take nonce also optional and pass nonce if exists. how do you think?